articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -33,11 +33,22 @@ You must also have access to either:
 
 Squid proxy is installed as a service on the control plane load balancers. Within the Squid configuration, we have specified an ACL (Access Control List) as being a list of destination domains (present in a file on the file system), which we will refer to as the whitelist for the remainder of this article. Outbound traffic from the cluster's internal network will pass through the proxy with the destination domain being compared against the whitelist for each connection attempt; if the domain exists then outbound internet traffic is permitted, otherwise it is denied with a HTTP 403 error.
 
+By default, the following domains are added to the whitelist (and cannot be removed) to facilitate installation, testing and ongoing operation of the cluster:
```

Comment:
This is good - but actually we should really rationalise this list in our code (and then this documentation too) - there shouldn't be a need for docker, rubygems or github except for our testing, and its perfectly reasonable for us to add these, test and then remove. I'd also like us to be more specific about the ukcloud one and make it the specific endpoints. If the customer wants to open those up, then that's within their power - but opening up all of github and docker by default and the customer not being able to remove them doesn't seem justifiable.

Reply:
Rationalised domains (having tested a deployment with a reduced number) and removed non-necessary ones